integration/resource_lacework_integration_aws_eks_audit_log_test.go

```diff
@@ -17,9 +17,10 @@ func TestIntegrationAwsEksAuditLog(t *testing.T) {
 		TerraformDir: "../examples/resource_lacework_integration_aws_eks_audit_log",
 		EnvVars:      tokenEnvVar,
 		Vars: map[string]interface{}{
-			"role_arn":    "arn:aws:iam::249446771485:role/lacework-iam-example-role",
-			"external_id": "12345",
-			"sns_arn":     "arn:aws:sns:us-west-2:123456789123:foo-lacework-eks",
+			"role_arn":      "arn:aws:iam::249446771485:role/lacework-iam-example-role",
+			"external_id":   "12345",
+			"sns_arn":       "arn:aws:sns:us-west-2:123456789123:foo-lacework-eks",
+			"s3_bucket_arn": "arn:aws:s3:::example-bucket-name",
```

Comment:
Think there might be an issue when the new optional field is not supplied.

We might want to add another integration test without this field to confirm

Reply:
Good point